### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by shuklanirdesh82**
Adding uninstallation instruction before installing managed plugin

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -49,10 +49,19 @@ esxcli software vib install -v /tmp/<vib_name>.vib
 Make sure you provide the **absolute path** to the `.vib` file or the install will fail.
 
 ### On Docker Host (VM)
+#### Managed Plugin
+Please make sure to uninstall older releases of DEB/RPM if installed using following commands.
```
Comment (pdhamdhere) [inline README.md]: Remove "if installed"

File: README.md [hunk 2]
```diff
@@ -49,10 +49,19 @@ esxcli software vib install -v /tmp/<vib_name>.vib
...
+```
+-sudo dpkg -P docker-volume-vsphere # Ubuntu or deb based distros
```
Comment (pdhamdhere) [inline README.md]: Why "-P" and not "-r"? As per documentation "-P" removes conf files as well. Managed plugin uses same file.

File: README.md [hunk 3]
```diff
@@ -49,10 +49,19 @@ esxcli software vib install -v /tmp/<vib_name>.vib
...
+-sudo rpm -e docker-volume-vsphere # Photon or rpm based distros
```
Comment (pdhamdhere) [inline README.md]: Remove "-" prefix from commands here and other places.

Comment (shuklanirdesh82) [inline README.md]: done!

File: README.md (hunk 1 above)
Comment (shuklanirdesh82) [inline README.md]: Done!

File: README.md (hunk 2 above)
Comment (shuklanirdesh82) [inline README.md]: I see, make sense.

**Pull request merged**